k8s/templates/router.yaml

```diff
@@ -70,7 +70,7 @@ metadata:
   labels:
     app: internal-router
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
Can we template this like for ingress https://github.com/mozilla-iam/dino-park-fence/blob/089628fa32799e155f809dc15661236f3345c024/k8s/values/prod.yaml#L8

Reply:
makes sense. I've use the occasion to add it to all the deployments